test/integration/backward_compatible/parallel/ringbuffer/RingbufferProxyTest.js

```diff
@@ -158,4 +158,13 @@ describe('RingbufferProxyTest', function () {
         const size = await rb.size();
         expect(size.toNumber()).to.equal(2);
     });
+
+    it('readMany correctly works with a filter', async function () {
```

Comment:
Is this change related?

Reply:
Not related. I dont know, how it was included in this PR? I will fix it.